src/types.ts

```diff
@@ -1,4 +1,5 @@
 export type Numberish = bigint | number | string | Dnum;
 export type Value = bigint;
 export type Decimals = number;
+export type Rounding = "ROUND_HALF" | "ROUND_UP" | "ROUND_DOWN";
```

Comment:
Let’s also export `Rounding` from `src/index.ts`, so consumers can use it if needed

Reply:
Yes good idea